src/story.test.ts

```diff
@@ -67,12 +67,23 @@ const CSF3Story: XStory = {
 };
 
 const CSF3StoryStrict: XStory<ButtonArgs> = {
-  render: (args) => 'Named Story',
+  render: args => 'Named Story',
   name: 'Another name for story',
   decorators: [storyFn => `Wrapped(${storyFn()}`],
   parameters: { a: [1, '2', {}], b: undefined, c: Button },
   loaders: [() => Promise.resolve({ d: '3' })],
   args: { x: '1' },
+  play: ({ step }) => {
+    step('a step', ({ step: substep }) => {
+      substep('a substep', () => {});
+    });
+  },
+};
+
+const project: ProjectAnnotations<XFramework> = {
+  async runStep(stepFn, context) {
+    return stepFn(context);
+  },
```

Comment:
This will be the default implementation in the preview.

Reply:
Shouldn't the first argument be the label?

Reply:
Oh, I guess so, yes I'll add that.

Reply:
It needs a label as first argument, right?